cpe-remap.yaml

```diff
@@ -2,7 +2,6 @@ mappings:
   alpine:
     vendor: alpinelinux
   apache:
-    vendor: apache
     products:
```

Comment:
The new changes make entries mapping the vendor back to the same string unnecessary.